bolt-cli/src/cli.rs

```diff
@@ -63,6 +64,28 @@ impl Cmd {
             Self::PubkeyHash(cmd) => cmd.run(),
         }
     }
+
+    /// Asks whether the user wants to proceed further. If not, the process is exited.
+    #[allow(unreachable_code)]
+    pub fn request_confirmation() {
+        // Skip confirmation in tests
+        #[cfg(test)]
+        return;
+
+        inquire::Confirm::new("Do you want to continue? (yes/no):")
+            .prompt()
+            .map(|proceed| {
+                if proceed {
+                    return;
+                }
+                info!("Aborting");
+                std::process::exit(0);
+            })
+            .unwrap_or_else(|err| {
+                info!("confirmation exited: {}", err);
+                std::process::exit(0);
+            })
+    }
```

Comment:
this is awesome, however could you move this function back in bolt-cli/src/common/mod.rs? 
We usually avoid using static methods. 

Thanks for the contribution!